### PR TITLE
Arc VM Feedback 01-29

### PR DESCRIPTION
**Pull request by dansisson**
If this PR needs approval, or if the commits need to be pulled into the private repo, contact @ManikaDhiman. Thanks.

### CODE REVIEW

**Comment by dansisson** [comment]
If this PR needs approval, or if the commits need to be pulled into the private repo, contact @ManikaDhiman. Thanks.

**Comment by Court72** [comment]
@ManikaDhiman 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-stack/includes/manage-arc-vm-2.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+author: alkohli
+ms.topic: include
+ms.date: 01/29/2024
+ms.author: alkohli
+
+---
+
+> [!NOTE]
```
Comment (ManikaDhiman) [inline azure-stack/includes/manage-arc-vm-2.md:9]: @dansisson: Remove '> > [!NOTE]' from line#10.

File: azure-stack/includes/manage-arc-vm.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+author: alkohli
+ms.topic: include
+ms.date: 01/29/2024
+ms.author: alkohli
+
+---
+
+> [!NOTE]
+> The recommended way to create and manage VMs on Azure Stack HCI 23H2 is using the [Azure Arc control plane](/hci/manage/azure-arc-vm-management-overview.md). Use the mechanism described below to manage your VMs only if you need functionality that is not available in Azure Arc VMs.
```
Comment (ManikaDhiman) [inline azure-stack/includes/manage-arc-vm.md:10]: Replace 'Azure Stack HCI 23H2' with 'Azure Stack HCI, version 23H2'

**Review by ManikaDhiman — COMMENTED** [review]
Reviewed the PR and provided my comments. Thanks!

**Comment by dansisson** [comment]
This is a working PR that will be added to over time. This is just the first wave of changes - thanks!

**Pull request merged**